src/test/kotlin/com/atlassian/performance/tools/report/api/DataReporterTest.kt

```diff
@@ -49,7 +50,8 @@ class DataReporterTest {
 
     @Test
     fun shouldAggregateAverage() {
-        val report = workDir.newFile()
+        val report = Paths.get("build").resolve("test.html").toFile()
+        report.createNewFile()
```

Comment:
Why are we changing the `DataReporterTest`?

Reply:
This is a leftover. I will not touch this test